### PR TITLE
hobbies.json: Add two more addiction-based hobbies

### PR DESCRIPTION
**Pull request by NetSysFire**
...and fixed a typo.

#### Summary
Content "Add starting hobbies featuring a caffeine and sleeping pill addiction"

#### Purpose of change

There are a few hobbies which lets you start with an addiction, but it was not possible to start with a caffeine and sleeping pill addiction, which should be both "reasonably" common.
This also fixes a typo, but it is so minor that it is not worth mentioning.
The only addiction now missing is a cocaine one, but since there is already a crack-cocaine one, I am not too sure about adding one.

#### Describe the solution

I basically copied the Alcoholic hobby and pasted it two times and adapted the description and the addiction type.

#### Describe alternatives you've considered

n/a

#### Testing

Started a game, chose those hobbies, waited two hours ingame and the withdrawal kicks in as expected. I'd like some feedback on the intensity I chose for the addictions.

#### Additional context

n/a

### CODE REVIEW

File: data/json/hobbies.json [hunk 1]
```diff
@@ -56,6 +56,24 @@
     "traits": [ "DISORGANIZED" ],
     "addictions": [ { "intensity": 20, "type": "alcohol" } ]
   },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "sleepingpillhead",
+    "name": "Sleeping Pill Reliance",
+    "description": "You had difficulties falling asleep, so you started taking those sleeping pills. Problem is, now you can not fall asleep without them anymore and the Cataclysm does complicate the matter.",
```
Comment (BrettDong) [inline data/json/hobbies.json:64]:
```suggestion
    "description": "You had difficulties falling asleep, so you started taking those sleeping pills.  Problem is, now you can not fall asleep without them anymore and the Cataclysm does complicate the matter.",
```

File: data/json/hobbies.json [hunk 2]
```diff
@@ -56,6 +56,24 @@
...
+    "points": -1,
+    "addictions": [ { "intensity": 10, "type": "sleeping pill" } ]
+  },
+  {
+    "type": "profession",
+    "subtype": "hobby",
+    "id": "caffeine_junkie",
+    "name": "Caffeine Junkie",
+    "description": "Coffee was your day-to-day companion in surviving your job, but getting caffeine into your body will be much harder now.",
+    "points": -1,
```
Comment (Termineitor244) [inline data/json/hobbies.json:74]:
```suggestion
    "points": -1,
    "traits": [ "STIMBOOST" ],
```

Comment (Termineitor244) [reply data/json/hobbies.json:74]: This trait was basically created for caffeine addicts, and its benefits are mainly for the use of caffeine, so it makes sense to have it with the caffeine addict.

**Comment by faefux** [comment]
I feel like i made this point clearly. You would not call an addiction a 'hobby' the same as you would not call bulimia a hobby or self harm a hobby. It is out of date thinking that belongs firmly in the past and you do yourself little credit depicting it as such.

**Comment by Termineitor244** [comment]
> 
> 
> I feel like i made this point clearly. You would not call an addiction a 'hobby' the same as you would not call bulimia a hobby or self harm a hobby. It is out of date thinking that belongs firmly in the past and you do yourself little credit depicting it as such.

I feel this it out of scope, but I concord with you, maybe renaming the feature to something like "Background" would fit better, but that should be addressed in a separated issue.

**Pull request merged**